theia-flutter-docker/latest.package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia Flutter Example",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/editor-preview": "latest",
+        "@theia/file-search": "latest",
+        "@theia/getting-started": "latest",
+        "@theia/git": "latest",
+        "@theia/json": "latest",
+        "@theia/markers": "latest",
+        "@theia/messages": "latest",
+        "@theia/navigator": "latest",
+        "@theia/outline-view": "latest",
+        "@theia/plugin-ext-vscode": "latest",
+        "@theia/preferences": "latest",
+        "@theia/preview": "latest",
+        "@theia/search-in-workspace": "latest",
+        "@theia/terminal": "latest",
+        "@theia/textmate-grammars": "latest"
+    },
+    "devDependencies": {
+        "@theia/cli": "latest",
+        "@theia/debug": "latest"
+    },
+    "resolutions": {
+        "vscode-languageserver-protocol": "3.15.0-next.9",
+        "vscode-languageserver-types": "3.15.0-next.5",
+        "**/vscode-json-languageserver/**/vscode-languageserver": "6.0.0-next.1"
+    },
+    "scripts": {
+        "postinstall": "download-debug-adapters"
+    },
+    "adapterDir": "plugins",
+    "adapters": {
```

Comment:
Do you mind following the newer examples such as `theia-cpp` which uses `theiaPlugins` to list the plugins which should be added to the application, and instead make use of the `download:plugins` script.

https://github.com/theia-ide/theia-apps/blob/f72385a3cc8d2786dc4b3d7b877fe0b5f576b6e1/theia-cpp-docker/latest.package.json#L66